cypress/global.d.ts

```diff
@@ -96,6 +96,11 @@ declare namespace Cypress {
      */
     loginByApi(username: string, password?: string): Chainable<Response>;
 
+    /**
+     * Logs-in user by using Okta API request
+     */
+    loginByOktaApi(username: string, password?: string): Chainable<Response>;
```

Comment:
Sorry, this is probably not the correct place to ask this question. But I have a project that's using the same setup just like this (cypress-real-world-project) and run into a situation where I need to import some typings from a source file (types that are used in src dir) but I'm not sure how to reference them from this file.

Reply:
@konekoya Are you describing the [TypeScript Triple-Slash Directive](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html).  We use it at the top of this file.

Reply:
As far as I know, that triple-slash directive is for importing another `.d.ts` file, right? 

I'm not sure if this is the correct way to do it. But we need to reference a type which lives in another file, I will use the code here as the example

Import a type from another file under Cypress namesapce
```js
import { User } from "../src/models";
```

Add a new `createUser` type definition and use that `User` type, as you can see, although the `User` is imported and properly typed, the `user` parameter is still getting `any` as the type
![Type definition in global.d.ts](https://imgur.com/hiHztQU.png)

And when using it, Vscode is also suggesting that parameter user should be a type of `any`
![Use in the spec](https://imgur.com/S8WXQnu.png)

I even tried to type the custom command directly, but still, the `user` parameter is still typed as `any`
![Custom command](https://imgur.com/u7cG0VL.png)

Hope this clearly illustrates the issue and thanks for the help 🙏